### PR TITLE
Announcing campaign with Triplequote

### PR DESCRIPTION
**Pull request by kailuowang**
A blog post to announce this program
cc @milessabin @larsrh

### CODE REVIEW

File: posts/2019-05-29-support-typelevel-thanks-to-triplequote-hydra.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+Using Hydra has proven to be incredibly valuable for Scala projects that rely on our ecosystem, and it delivered impressive compilation speedups. Nowadays, even [Cats is compiled with Hydra](https://github.com/typelevel/cats/pull/2848). Therefore, if you are looking for a way to help us reach our fundraising goal, while also profiting from a great product, don’t let this opportunity slip away. Head over to the [Hydra trial page](https://triplequote.com/hydra/trial) and get started with it in no time using your preferred development tool, whether that is sbt, Maven, Gradle, or IntelliJ IDEA!
+
+To sum it up, for each yearly Hydra license purchased online by June 30th, you get a 25% discount and an additional 25% is donated to the Typelevel Sustainability Program. Just don’t forget to use the discount code **TYPELEVEL25** ;-)
```
Comment (larsrh) [inline posts/2019-05-29-support-typelevel-thanks-to-triplequote-hydra.md]: I think this paragraph is redundant.

Comment (kailuowang) [reply posts/2019-05-29-support-typelevel-thanks-to-triplequote-hydra.md]: reduced to a one sentence reminder of the discount code.

Comment (larsrh) [reply posts/2019-05-29-support-typelevel-thanks-to-triplequote-hydra.md]: Please also remove the emoji.

Comment (kailuowang) [reply posts/2019-05-29-support-typelevel-thanks-to-triplequote-hydra.md]: done ;)

**Comment by larsrh** [comment]
:clap: :tada:

**Comment by larsrh** [comment]
I'll tweet out the announcement once it appears on the website.

**Comment by kailuowang** [comment]
Thanks very much!

**Pull request merged**